tests/schema/tests.py

```diff
@@ -2370,6 +2370,21 @@ def test_add_datefield_and_datetimefield_use_effective_default(self, mocked_date
             cast_function=lambda x: x.time(),
         )
 
+    def test_namespaced_db_table_create_index_name(self):
+        """
+        Table names should be stripped of their namespace/schema before being
+        used to generate index names.
+        """
+        with connection.schema_editor() as editor:
+            max_name_length = connection.ops.max_name_length() or 200
+            namespace = 'n' * max_name_length
+            table_name = 't' * max_name_length
+            namespaced_table_name = '%s"."%s' % (namespace, table_name)
```

Comment:
I think that leading and trailing `"` is missing, i.e. `'"%s"."%s"' % (namespace, table_name)`.

Reply:
@felixxm `split_identifier` is be able to deal with both cases but I'll adjust the test, thanks!